src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatXMLWriter.cs

```diff
@@ -37,7 +38,7 @@ internal class FormatXmlWriter
             StreamWriter streamWriter;
             FileStream fileStream;
             FileInfo fileInfo;
-            PathUtils.MasterStreamOpen(cmdlet, filepath, "ascii", true, false, force, noclobber,
+            PathUtils.MasterStreamOpen(cmdlet, filepath, PowerShellEncoding.GetEncoding(FileEncoding.Ascii), true, false, force, noclobber,
```

Comment:
This appears to be the only external call to Encoding overload of PathUtils.MasterStreamOpen.  Suggest using FileEncoding.Ascii here and merge the two MasterStreamOpen overloads.